### PR TITLE
Fix image scaling and restore list rendering in blog posts

### PR DESCRIPTION
**Pull request by saumyashahi**
## Summary:

- Fix image rendering and bullet point display issues in the posts.
- Fixes: #622

## Key Changes:

- Made images responsive by adding max-width: 100%.
- Updated list styles to restore bullet points
- Validated markdown formatting to ensure proper rendering in Jekyll.

## Testing:

- Build the site locally with jekyll serve and verify image and list rendering.
- Check bullet points and image scaling on mobile and desktop views.

### CODE REVIEW

**Comment by quozl** [comment]
Next time please update the existing pull request rather than close and start a new one.  We are all notified less that way.

File: css/custom.css [hunk 1]
```diff
@@ -281,3 +281,8 @@
   display: none;
 }
 
+/* fix list rendering */
+ul {
+  padding: revert;
+  list-style: inherit;
+}
```
Comment (quozl) [inline css/custom.css:288]: Missing newline at end of file.

File: _posts/2024-12-29-sweet-spot-002.md [hunk 2]
```diff
@@ -62,6 +62,7 @@ donation or purchasing merchandise, and please help spread the word.
 * See our new donation banner, created by volunteers in our development community: <https://www.sugarlabs.org/>   
 * Contribute to Sugar Labs by purchasing Sugar Labs brand merchandise: <https://www.bonfire.com/sugar-labs-education/>
 
+^
```
Comment (quozl) [inline _posts/2024-12-29-sweet-spot-002.md:65]: An explanation of `^` would be welcome ... in the commit message.

**Review by quozl — COMMENTED** [review]
* [ ] commit message too brief "Fix list rendering" doesn't explain anything, please rewrite your commit messages; https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits

**Comment by saumyashahi** [comment]
@pikurasa asked me to open a new pr with changes in custom.css file rather than airspace.css. 

> Next time please update the existing pull request rather than close and start a new one. We are all notified less that way.

I will keep in mind with all further PRs, and do the required changes here.

File: css/custom.css (hunk 1 above)
Comment (saumyashahi) [reply css/custom.css]: I have fixed this 👍

File: _posts/2024-12-29-sweet-spot-002.md (hunk 2 above)
Comment (saumyashahi) [reply _posts/2024-12-29-sweet-spot-002.md:65]: Sure, I have edited the commit message accordingly.

**Comment by pikurasa** [comment]
@saumyashahi It's a relief to have this. Thank you.

**Comment by saumyashahi** [comment]
> @saumyashahi It's a relief to have this. Thank you.

My pleasure to have helped :)

**Pull request merged**